src/locales/de.json

```diff
@@ -136,6 +136,13 @@
 "ech": "Geokatalog",
 "ech_service_link_href": "https://www.geo.admin.ch/de/home.html",
 "ech_service_link_label": "geo.admin.ch",
+"edit_back": "Zurück / Editieren beenden",
+"edit_file_saved": "Alle Änderungen gespeichert",
+"edit_file_saving": "Speichern ....",
+"edit_mode_title": "",
+"edit_panel_title": "",
+"edit_reset": "",
```

Comment:
just use the english ones until we have somehing, it looks weird otherwise.

Reply:
It comes from the google sheets they will put what they want. An it warns them that they need to translate.